types/trigger.d.ts

```diff
@@ -55,3 +55,16 @@ export type ResponseOutput<Matches> = {
 // The type of a response trigger field.
 export type ResponseFunc<Matches> =
     (data: Data, matches: Matches, output: Output) => ResponseOutput<Matches>;
+
+export type Trigger = {
```

Comment:
I think `beforeSeconds` is only for timeline triggers.  Maybe call this `TimelineTrigger` instead?